python/bundle_bootstrap.py

```diff
@@ -0,0 +1,255 @@
+#!/usr/bin/env python3
+
+# Dependencies:
+# - DC/OS CLI (specified in requirement.txt)
```

Comment:
```suggestion
# - DC/OS CLI (specified in Pipfile)
```
Can you please also update this in other places?

Reply:
done